docs/content/modeling/contextual-time-based-authorization.mdx

```diff
@@ -877,7 +877,7 @@ When Anne is connecting from a denied ip address range or timeslot, <ProductName
 Contextual tuples:
 
 - Are not persisted in the store.
-- Are only supported on the <UpdateProductNameInLinks link="/api/service#Relationship%20Queries/Check" name="Check API endpoint" /> and <UpdateProductNameInLinks link="/api/service#Relationship%20Queries/ListObjects" name="ListObjects API endpoint" />. They are not supported on read, expand and other endpoints.- If you are using the [ReadChanges API endpoint](/api/service#Relationship%20Tuples/ReadChanges) to build a permission aware search index, note that it will not be trivial to take contextual tuples into account.
+- Are only supported on the <UpdateProductNameInLinks link="/api/service#Relationship%20Queries/Check" name="Check API endpoint" /> and <UpdateProductNameInLinks link="/api/service#Relationship%20Queries/ListObjects" name="ListObjects API endpoint" />. They are not supported on read, expand and other endpoints.
```

Comment:
Any reason we're removing this part?

Reply:
I see it duplicated, it is the next item